apps/src/lab2/views/dialogs/extractDialogClosePromiseInput.ts

```diff
@@ -0,0 +1,15 @@
+import {DialogClosePromiseReturnType} from '@cdo/apps/lab2/views/dialogs';
+
+// given a promise returned from DialogManager.showDialog({type : DialogType.GenericPrompt}), will return the input
+// that was typed in by the user.
+// Note that if the user did not press the `confirm` button, then an empty string will be returned instead.
+export const extractDialogClosePromiseInput = (
```

Comment:
is this used anywhere?

Reply:
No it is not. I thought I had `git mv'ed` that file into `lab2/views/dialogs/extractUserInput` but clearly had not. Deleted it.